guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -62,6 +83,20 @@ To add an image, complete the following steps:
 . Optional: If the selected image supports `cloud-init`, select the *User data* check box to enable user data input.
 . Click *Submit* to save the image details.
 
+.For CLI Users
+
+To add an image, enter the following command. With the `--username` option, specify a user other than `root`, because the `root` user cannot connect to a GCE instance using SSH keys. The username must begin with a letter and consist of lowercase letters and numbers.
```

Comment:
Please be consistent with the way you refer to the command. In line 56, you use one sentence structure, and use another now. It is better if we are consistent.